src/web/modules/gateway_accounts/csv_data.ts

```diff
@@ -65,7 +65,7 @@ export async function createCsvWithAdminEmailsData(filters: Filters): Promise<an
         payment_email_enabled: account.email_notifications['PAYMENT_CONFIRMED'] && account.email_notifications['PAYMENT_CONFIRMED'].enabled || false,
         refund_email_emailed: account.email_notifications['REFUND_ISSUED'] && account.email_notifications['REFUND_ISSUED'].enabled || false,
         custom_branding: service.custom_branding !== undefined,
-        email_branding: account.notify_settings !== undefined,
+        email_branding: account.notifySettings !== undefined,
```

Comment:
You might question whether this breaks anything that was working - it doesn't as the notify settings are never returned on the search gateway accounts results from connector. Don't think this ever worked - it should probably be removed, but I'll do that separately.